src/feature/signup/components/TermsModal.tsx

```
@@ -0,0 +1,126 @@
+          {term.subTitle && <h2 className="b4 leading-[26px]">{term.subTitle}</h2>}
+
+          {term.section.map((section, idx) => (
+            <div key={idx} className="flex flex-col gap-[20px]">
+              <h2 className="h7 text-primary-color">{section.heading}</h2>
+
+              {Array.isArray(section.content) ? (
+                <ul className="flex flex-col gap-[15px]">
+                  {section.content.map((line) => (
+                    <li className="b4 leading-[26px]">{line}</li>
+                  ))}
+                </ul>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)